apps/projects/routes.py

```diff
@@ -913,6 +924,38 @@ class RetrieveOrCreateThumbnails(MethodView):
             'type': 'float',
             'coerce': float,
         },
+        'crop': {
+            'type': 'dict',
+            'coerce': literal_eval,  # crop args are a string represent of a dict
```

Comment:
Note that use of `literal_eval` to coerce here is safe (https://bugs.python.org/msg312701), because value pass to `literal_eval` always is strings and limit of maximum size of `GET` request (4KB / 8KB)